### DISCUSSION TITLE
Hugo warnings

### DISCUSSION THREAD

**Discussion by sagarbehere** [razonyang/hugo-theme-bootstrap#529] [Q&A] [answered]
I have just upgraded this theme from v0.67.1 to v0.79.1 and am now using Hugo v0.99.1 instead of v0.92. After these changes, I am seeing the following message after running `hugo server` and also in the Netlify console while it is building my site

```
WARNING: calling IsSet with unsupported type "bool" (bool) will always return false.
```
Running `grep -r "isset" themes/hugo-theme-bootstrap` shows the files are using the `isset` function
```
themes/hugo-theme-bootstrap/layouts/shortcodes/neteasemusic.html:{{- if isset .Params 1 -}}
themes/hugo-theme-bootstrap/layouts/shortcodes/neteasemusic.html:{{- if isset .Params 2 -}}
themes/hugo-theme-bootstrap/layouts/partials/post/copyright/index.html:        {{- if isset .Site.Params "creativecommons" -}}
themes/hugo-theme-bootstrap/layouts/partials/header/settings/font-sizes.html:{{ if isset $.Site.Params "fontsize" }}
themes/hugo-theme-bootstrap/layouts/partials/assets/service-worker/js.html:{{- if isset $.Site.Params "pwa" -}}
themes/hugo-theme-bootstrap/layouts/partials/helpers/fullwidth.html:{{- else if isset .Page.Site.Params.fullWidth $section -}}
themes/hugo-theme-bootstrap/layouts/partials/head/meta/robots.html:{{- if and (not (and (isset .Params "metarobots") (not .Params.metaRobots))) $content -}}
themes/hugo-theme-bootstrap/assets/main/scss/_config.scss:{{- if isset $params "palette" -}}
```
I can confirm that no file in the `layouts` folder in my site root is using `isset`.  

1. Has anyone else seen this WARNING? Despite the warning, everything _seems_ to be working on my site. I'm just curious if this is pointing to a potential hidden bug.
2. Does anyone here know how to figure out which file/line is causing the WARNING? I searched a bit, but it seems that Hugo does not provide more details on where the warning is coming from.

**Comment by razonyang** [comment, marked as answer]
It's relative to the `fullWidth` parameter, it's not a potential bug, you can ignore this `WARNING` message.
